content/blog/multi-stage-yaml-pipeline-to-create-and-publish-nuget-packages.md

```diff
@@ -0,0 +1,263 @@
+
+That's where the second step comes in. The [DownloadPipelineArtifact](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/download-pipeline-artifact?view=azure-devops) task will download any artifacts that were published in the pipeline at an earlier stage. They will be available in the `$(Pipeline.Workspace)` folder.
+
+The [NuGetCommand](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/package/nuget?view=azure-devops) task will publish the prerelease package to an internal Azure DevOps Artifacts feed called 'Test'.
+
+That's everything you need to publish the prerelease package. You can give the pipeline another run to see if it works.
+
+### Publish release package to nuget.org
+
+The last stage will publish the release version of the NuGet package to nuget.org. But before we proceed we'll need to make a few preparations.
+
+#### Create new environment
+
+In the 'old style' release pipelines, approvals were configured on stages itself. With the new multi-stage pipelines this has been moved to environments. So, we'll have to create an environment first.
+
+- Click the _Environments_ menu item in the Azure DevOps portal.  
+  (When you enabled the _Multi-stage pipelines_ preview feature this new menu item appeared in the _Pipelines_ menu.)
+- Choose _New environment_.
+- Specify a name like 'nuget-org' (a . is not allowed) and an optional description.  
+![New environment](../../static/images/multi-stage-yaml-pipeline-to-create-and-publish-nuget-packages/new-environment.png)
+<!-- ![New environment](../../../../../images/multi-stage-yaml-pipeline-to-create-and-publish-nuget-packages/new-environment.png) -->
+- Click Create.
+
+After the environment is created, open it to configure users or groups to give their approval before the package is published.
+
+- Click on the button with three dots.
+- Choose _Checks_.  
+![Environment add checks](../../static/images/multi-stage-yaml-pipeline-to-create-and-publish-nuget-packages/environment-add-checks.png)
+<!-- ![Environment add checks](../../../../../images/multi-stage-yaml-pipeline-to-create-and-publish-nuget-packages/environment-add-checks.png) -->
+- Click _Create_ in the new screen.
+- Add users and groups and specify optional instructions for the approvers.  
+![Create approvals](../../static/images/multi-stage-yaml-pipeline-to-create-and-publish-nuget-packages/environment-add-checks-create-approvals.png)
+<!-- ![Create approvals](../../../../../images/multi-stage-yaml-pipeline-to-create-and-publish-nuget-packages/environment-add-checks-create-approvals.png) -->
+- Click _Create_.
+
+The environment is now ready to deploy to.
+
+#### Configure nuget.org service connection
+
+  To be able to push a package to nuget.org you'll need an account. Create an account on [nuget.org](https://www.nuget.org) if you haven't already and log in. Then create an api key as described [here](https://docs.microsoft.com/en-us/nuget/nuget-org/scoped-api-keys).
```

Comment:
Spatie voor de zin :P

Reply:
Ik zou een komma toevoegen aan de volgende zin: Create an account, if you haven't already, on [nuget.org] and login.